### ISSUE TITLE
Async usage of the API makes the server crash

### ISSUE THREAD

**Issue by ZeTioZ** [ryandw11/UltraBar#45]
Hi,

I recently tried to use your API to make some boss bar. I've noticed that your plugin does make the server crash if you use the api in async. Maybe you could add a little warning in the api documentation to not use it async and exclusively use it sync to avoid issues like that.

To reproduce that, you just have to create a bossbar with your API and then delete it in async. And if you do it a lot quite some time, it can crash the server saying that there is an unexisting entity ticked inside a chunk (which I guess is the EnderDragon). The stack trace would not even help you since it's a crash on the server core side, exactly on the chunk ticking system.

I'm on Spigot 1.19.2 and it does also occur on Paper 1.19.2.

Here is the error if you want to take a look at it: https://pastebin.com/ithdKiMn

Regards,
ZeTioZ

**Comment by ryandw11** [comment]
Correct, almost none of the bukkit api is thread safe so the same goes with my plugin. I'll add the warning to the documentation.
